vhdbuilder/packer/vhd-image-builder-mariner2.json

```diff
@@ -0,0 +1,391 @@
+{
```

Comment:
I don't think we need to add a new image builder json for gen1, HYPERV_GENERATION is passed in by the caller and there's no delta between mariner2 and mariner2-gen2 jsons. I'd just rename the existing one to vhd-image-builder-mariner2.json